### DISCUSSION TITLE
Hidden visible for admin

### DISCUSSION THREAD

**Discussion by utamayo** [GitbookIO/community#719] [Feature Requests]
When pages or articles on my Gitbook are hidden they are hide until you request a change. That is ok for user with read rights... but for admin or write rights is a little bit annoying because you don't see clear the contents which are hidden and if don't remember quite well you have to start a change request even if you are not going to change anything.

**Comment by SamyPesse** [comment]
This is done by design. We want the locked-mode / read view to represent as best as possible what end readers will experience. Hidden pages are designed to be hidden in all reading experiences and only visible in editable contexts where you need to be able to act on them.
